test/runtests.jl

```diff
@@ -233,6 +233,18 @@ x7   0.0914971  0.0286485   3.19378     0.0014
         -0.0848711  0.195757   -0.433554  0.6646;
          0.0914971  0.0286485   3.19378   0.0014
     ]
+    
+    # expected wald intervals from the coxph 
+    # function in the survival package in R
```

Comment:
Could you add the R code you'd use to get this in the comment? I believe the Kaplan-Meier tests further up in this file will provide an example of what I mean.